src/main/java/ubc/pavlab/rdp/controllers/LoginController.java

```diff
@@ -42,6 +48,27 @@ public class LoginController {
     @Autowired
     private ApplicationSettings applicationSettings;
 
+    @Autowired
+    private EmailValidator emailValidator;
+
+    @Autowired
+    private RecaptchaValidator recaptchaValidator;
+
+    @Autowired
+    private MessageSource messageSource;
+
+    @InitBinder
+    public void addEmailValidator( WebDataBinder dataBinder ) {
+        dataBinder.setAllowedFields( "user.email" );
+        dataBinder.addValidators( emailValidator );
+    }
+
+    @InitBinder("registerNewUser")
+    public void addRecaptchaValidator( WebDataBinder dataBinder ) {
```

Comment:
It would be nice to apply this more generally such that any place where a `Recaptcha` is used with a `@Valid` annotation, a check is done.